packages/git-butler-deb/git-butler-deb.pacscript

```diff
@@ -0,0 +1,14 @@
+name="git-butler-deb"
+# there is no repology property, since the package is not even in repology (with the exception of an AUR package, with a name that doesnt match the official deb file
+arch=('amd64')
+pkgver="0.10.6-619"
+homepage='https://gitbutler.com/'
+url="https://releases.gitbutler.com/releases/release/${pkgver}/linux/x86_64/git-butler_${pkgver%-*}_amd64.deb"
+depends=("libayatana-appindicator3-1" "libwebkit2gtk-4.0-37" "libgtk-3-0")
+breaks=("git-butler-git" "git-butler-bin")
+replace=("git-butler")
+gives="git-butler"
```

Comment:
Move this below `name`.

Reply:
I should just move the gives property right?

The replace and breaks properties are in a good position already?

Reply:
Just move `gives` below `name`.

Reply:
Done